kview/src/main/java/dev/ikm/komet/kview/mvvm/view/properties/StampAddController.java

```diff
@@ -0,0 +1,142 @@
+        initStatusComboBox();
+    }
+
+    private void initStatusComboBox() {
+        ObservableList<State> statuses = stampViewModel.getObservableList(STATUSES);
+        statuses.addListener((ListChangeListener<State>) c -> {
+            List<String> statusesStrings = statuses.stream()
+                                                   .map(this::toFirstLetterCapitalized)
+                                                   .collect(Collectors.toList());
+            Collections.sort(statusesStrings, NaturalOrder.getObjectComparator());
+            statusComboBox.getItems().setAll(statusesStrings);
+        });
+
+        ObjectProperty<State> statusProperty = stampViewModel.getProperty(STATUS);
+        statusProperty.subscribe(state -> {
+            if (state != null) {
+                statusComboBox.setValue(toFirstLetterCapitalized(state));
+            }
+        });
+    }
+
+    private void initPathComboBox() {
+        ObservableList<ConceptEntity> paths = stampViewModel.getObservableList(PATHS);
+        paths.addListener((ListChangeListener<ConceptEntity>) c -> {
+            List<String> pathStrings = paths.stream()
+                    .map(EntityFacade::description)
+                    .collect(Collectors.toList());
+            Collections.sort(pathStrings, NaturalOrder.getObjectComparator());
+            pathComboBox.getItems().setAll(pathStrings);
+        });
+
+        ObjectProperty<ConceptEntity> pathProperty = stampViewModel.getProperty(PATH);
+        pathProperty.subscribe(conceptEntity -> {
+            if (conceptEntity != null) {
+                pathComboBox.setValue(conceptEntity.description());
+            }
+        });
+    }
+
+    private void initModuleComboBox() {
+        // populate modules
+        ObservableList<ConceptEntity> modules = stampViewModel.getObservableList(MODULES);
+
+        modules.addListener((ListChangeListener<ConceptEntity>) c -> {
+            List<String> moduleStrings = modules.stream()
+                    .map(EntityFacade::description)
+                    .collect(Collectors.toList());
+            Collections.sort(moduleStrings, NaturalOrder.getObjectComparator());
+            moduleComboBox.getItems().setAll(moduleStrings);
+        });
+
+        ObjectProperty<ConceptEntity> moduleProperty = stampViewModel.getProperty(MODULE);
+        moduleProperty.subscribe(conceptEntity -> {
+            if (conceptEntity != null) {
+                moduleComboBox.setValue(conceptEntity.description());
+            }
+        });
+    }
+
+    private String toFirstLetterCapitalized(State status) {
+        String statusString = status.toString();
+        return statusString.substring(0, 1).toUpperCase() + statusString.substring(1).toLowerCase();
+    }
+
+    @Override
+    public StampViewModel2 getViewModel() {
+        return stampViewModel;
+    }
+
+    @Override
+    public void updateView() {
+
+    }
+
+    @Override
+    public void clearView() {
+
+    }
```

Comment:
now that we are using mvvm we should remove the methods updateView() and clearView()